core/src/main/scala/cats/syntax/parallel.scala

```diff
@@ -258,14 +269,45 @@ final class ParallelApOps[M[_], A](private val ma: M[A]) extends AnyVal {
     Parallel.parReplicateA(n, ma)
 }
 
-final class ParallelApplyOps[M[_], A, B](private val mab: M[A => B]) extends AnyVal {
+final class ParallelApOps1[M[_], A](protected val ma: M[A]) extends AnyVal {
+  def parReplicateA(n: Int)(implicit P: Parallel[M]): M[List[A]] =
+    Parallel.parReplicateA(n, ma)
+}
```

Comment:
This is absolutely disputable, so feel free to share your thoughts)
I'm wondering, what are the advantages we get if create a separate `ParallelApOps1` rather than keep `ParallelApOps` only and hide and deprecate only those methods which should go to `NonEmptyParallelApOps`?

My concern here is that we will end up with 3 classes in result jar: `ParallelApOps`, `ParallelApOps1`,  `NonEmptyParallelApOps` comparing to just 2 classes (`ParallelApOps` and `NonEmptyParallelApOps`) otherwise. Unfortunately, we cannot reduce this amount to just 1 due to BC, but 2 seems better to me than 3 unless there are some advantages brought up.

Reply:
I was really just trying to mirror the other instances in this file, e.g. `ParallelTraversableOps`, but can do it with only 2 classes instead :)